app/channel_test.go

```diff
@@ -1898,3 +1898,25 @@ func TestClearChannelMembersCache(t *testing.T) {
 
 	th.App.ClearChannelMembersCache("channelID")
 }
+
+func TestGetMemberCountsByGroup(t *testing.T) {
+	th := SetupWithStoreMock(t)
+	defer th.TearDown()
+
+	mockStore := th.App.Srv().Store.(*mocks.Store)
+	mockChannelStore := mocks.ChannelStore{}
+	cmc := []*model.ChannelMemberCountByGroup{}
+	for i := 0; i < 5; i++ {
+		cmc = append(cmc, &model.ChannelMemberCountByGroup{
+			GroupId:                     model.NewId(),
+			ChannelMemberCount:          1,
+			ChannelMemberTimezonesCount: 1,
+		})
+	}
+	mockChannelStore.On("GetMemberCountsByGroup", "channelID", true).Return(cmc, nil)
+	mockStore.On("Channel").Return(&mockChannelStore)
+	resp, err := th.App.GetMemberCountsByGroup("channelID", true)
+	require.Nil(t, err)
+	require.NotNil(t, resp)
```

Comment:
Can we also verify the contents of the actual slice returned instead of just checking the length? That might be a stricter check.

Reply:
Done.

Reply:
Thanks, I meant to just use `require.ElementsMatch`. We don't need any of the slice iteration. I updated the fork myself to save you from the trouble.